safe/reportv4/test/test_impact_report.py

```diff
@@ -39,6 +40,19 @@ def fixtures_dir(cls, path):
         dirname = os.path.dirname(__file__)
         return os.path.join(dirname, 'fixtures', path)
 
+    def print_fields_name(self, layer):
+        """
+
+        :param layer:
+        :type layer: QgsVectorLayer
+        :return:
+        """
+        fields = layer.pendingFields()
+        field_names = [field.name() for field in fields]
+        for name in field_names:
+            print name
```

Comment:
ahah ;-)
If you want some fields, you can use 
`from safe.test.debug_helper import print_attribute_table` ;-)

I will add your helper in this file.